ipynbname/__init__.py

```diff
@@ -89,6 +90,9 @@ def name() -> str:
     """ Returns the short name of the notebook w/o the .ipynb extension,
         or raises a FileNotFoundError exception if it cannot be determined.
     """
+    ip = get_ipython()
+    if '__vsc_ipynb_file__' in ip.user_ns:
+        return os.path.basename(ip.user_ns['__vsc_ipynb_file__'])
```

Comment:
Hey! I came across this PR as we're also interested in VS Code support. This suggested change doesn't behave the same as the existing code. The function's docstring says to return the notebook name without the `.ipynb` extension, but the new logic would preserve and return it (e.g. as `"notebook_name.ipynb"`, not just `"notebook_name"`). This may create problems depending on how users use the returned value. Could you perhaps have a look at that? 🙂